src/sentry/static/sentry/app/views/groupMerged/mergedToolbar.jsx

```diff
@@ -59,7 +59,8 @@ const MergedToolbar = createReactClass({
 
     // only need eventId, not fingerprint
     let [baseEventId, targetEventId] = Array.from(entries).map(([, eventId]) => eventId);
-    ProjectActions.openDiffModal({
+
+    openDiffModal({
```

Comment:
Likely not related to your changes, but this modal doesn't work at all (nothing happens on click). Can you look into that?